### DISCUSSION TITLE
SUBMIT JOB COMPACT job taking too long (doing nothing?)

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#4646] [Q&A]
- We issued 'SUBMIT JOB COMPACT' command
- After about an hour, we issued another command to see the results 'SHOW JOB <Job_Id_From_Above_Step>'. See below result of SHOW JOB command
- On the node the 'COMPACT JOB' in RUNNING state, I executed below command which also seems in hung?
curl "http://10.200.74.5:19779/admin?space=MySpace&op=compact"
- Can someone explain why its running state more than an hour? How does it take normally?

```
"Job Id(TaskId)","Command(Dest)","Status","Start Time","Stop Time","Error Code"
"175","COMPACT","FAILED","2022-09-14T15:31:51.000000","2022-09-14T15:32:51.000000","E_UNKNOWN"
"0","nebula-cluster-storaged-2.nebula-cluster-storaged-headless.cohort-search.svc.cluster.local","FINISHED","2022-09-14T15:31:51.000000","2022-09-14T15:59:59.000000","SUCCEEDED"
"1","nebula-cluster-storaged-1.nebula-cluster-storaged-headless.cohort-search.svc.cluster.local","FINISHED","2022-09-14T15:31:51.000000","2022-09-14T16:06:43.000000","SUCCEEDED"
"2","nebula-cluster-storaged-0.nebula-cluster-storaged-headless.cohort-search.svc.cluster.local","RUNNING","2022-09-14T15:31:51.000000","_EMPTY_","E_UNKNOWN"
```

**Comment by wey-gu** [comment]
Could you please share logs from storage-0, please?
